### DISCUSSION TITLE
How to customize no search results on table?

### DISCUSSION THREAD

**Discussion by hazwan91** [filamentphp/filament#8420] [Help] [answered]
![Screenshot 2023-09-12 112019](https://github.com/filamentphp/filament/assets/97268905/352a29bf-c260-463d-9a70-98634de6b60b)

Let just say a user search for something that doesn't exist. A message got displayed but yet i don't like the word such as `Create a district to get started` and also the button to be put there.

If my understanding is correct, this part should be on List Pages. i try to see on the extended file which is the `ListRecord` but yet to find the desired function.

**Comment by danharrin** [comment, marked as answer]
Search "empty state" in the docs

**Comment by hazwan91** [comment]
Thanks. can't see that.
